content/en/docs/reference/feature-gates.md

```diff
@@ -221,6 +220,6 @@ Each feature gate is designed for enabling/disabling a specific feature:
   enables the usage of [`local`](/docs/concepts/storage/volumes/#local) volume
```

Comment:
Not your fault, but please also update all the links in this file to point to `/content/en/docs/` where they start with `/docs/`. This is due to the Hugo migration.